vendor/github.com/imdario/mergo/go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/imdario/mergo
+
+go 1.13
+
+require gopkg.in/yaml.v2 v2.3.0
```

Comment:
Was looking if we needed to update this one as well (currently we're using v2.2.8), but looks like the changes are not very relevant; https://github.com/go-yaml/yaml/compare/v2.2.8...v2.3.0